presto-main/src/test/java/com/facebook/presto/operator/scalar/FunctionAssertions.java

```diff
@@ -237,6 +240,18 @@ public void assertFunction(String projection, Type expectedType, Object expected
         }
     }
 
+    public void assertFunctionThrowsPrestoException(String projection, StandardErrorCode errorCode, String message)
+    {
+        try {
+            evaluateInvalid(projection);
+            fail("Expected to throw a PrestoException with message " + message);
+        }
+        catch (PrestoException e) {
+            assertEquals(e.getErrorCode(), errorCode.toErrorCode());
+            assertEquals(e.getMessage(), message);
```

Comment:
I think this is something that can be implemented when it will be needed first time.